docs/buildah-manifest-rm.md

```diff
@@ -0,0 +1,25 @@
+# buildah-manifest-rm "1" "April 2021" "buildah"
+
+## NAME
+buildah\-manifest-rm - Removes one or more manifest lists.
+
+## SYNOPSIS
+**buildah manifest-rm** [*list* ...]
+
+## DESCRIPTION
+Removes one or more locally stored manifest lists.
+
+## EXAMPLE
+
+buildah manifest rm listID
+
+buildah manifest-rm listID1 listID2
```

Comment:
```suggestion
buildah manifest rm listID1 listID2
```
and if that is indeed correct with the dash, could you add a test with that please?

Reply:
fixed